front/app/containers/InitiativesShow/ReactionControl/Status/ProposedReacted.tsx

```diff
@@ -99,18 +61,31 @@ const ProposedReacted = ({
   initiative,
   initiativeSettings: { reacting_threshold },
   onCancelReaction,
+  onReaction,
+  userReacted,
 }: StatusComponentProps) => {
   const reactionCount = initiative.attributes.likes_count;
   const reactionLimit = reacting_threshold;
   const daysLeft = getPeriodRemainingUntil(initiative.attributes.expires_at);
 
   return (
     <Container>
-      <StyledIcon ariaHidden name="check-circle" />
-      <ReactedTitle>
-        <FormattedMessage {...messages.votedTitle} />
-      </ReactedTitle>
-      <ReactedText>
+      <Box ml="auto">
+        <CountDown targetTime={initiative.attributes.expires_at} />
+      </Box>
+      <Box mb="8px">
+        <StyledIcon
+          fill={colors.success}
+          width="31px"
+          height="31px"
+          ariaHidden
+          name="check-circle"
+        />
+      </Box>
+      <Text m="0">
```

Comment:
Don't pay to much attention to this Text component. Still changes coming in next PRs.